test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_handlers_test.clj

```diff
@@ -301,4 +343,4 @@
                                                                 :follow-redirects false})]
           (is (= (:status response) 302))
           (is (= (get-in response [:headers "location"]) "http://localhost:8080/hello/"))
-          (is (= (get-in response [:opts :url]) "http://localhost:8080/hello")))))))
\ No newline at end of file
+          (is (= (get-in response [:opts :url]) "http://localhost:8080/hello")))))))
```

Comment:
You're reading it backwards.  Original file *lacked* a trailing newline.  My editor on save ensures there's a trailing newline, so we get this whitespace change.

If need be I'll remove this new trailing newline.